app/src/main/java/com/teamwss/websoso/ui/notice/NoticeActivity.kt

```diff
@@ -1,14 +1,49 @@
 package com.teamwss.websoso.ui.notice
 
 import android.os.Bundle
+import androidx.activity.viewModels
+import androidx.lifecycle.Observer
 import com.teamwss.websoso.R
 import com.teamwss.websoso.databinding.ActivityNoticeBinding
 import com.teamwss.websoso.ui.common.base.BindingActivity
+import com.teamwss.websoso.ui.main.MainActivity
 import dagger.hilt.android.AndroidEntryPoint
 
 @AndroidEntryPoint
 class NoticeActivity : BindingActivity<ActivityNoticeBinding>(R.layout.activity_notice) {
+
+    private val viewModel: NoticeViewModel by viewModels()
+
+    private val adapter: NoticeAdapter by lazy {
+        NoticeAdapter()
+    }
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+
+        setupClickListeners()
+        setupRecyclerView()
+        setupObservers()
```

Comment:
r: 
```suggestion
        setupObserver()
```

s 안 붙여도 되어요!